apiserver/pkg/manager/resource_manager.go

```diff
@@ -260,3 +264,28 @@ func getComputeTemplateByName(ctx context.Context, client clientv1.ConfigMapInte
 
 	return runtime, nil
 }
+
+func (r *ResourceManager) GetClusterEvents(ctx context.Context, clusterName string, namespace string) ([]v1.Event, error) {
+	client := r.getEventsClient(namespace)
+
+	clusterClient := r.getRayClusterClient(namespace)
+	return getRayClusterEventsByName(ctx, clusterName, client, clusterClient)
+}
+
+func getRayClusterEventsByName(ctx context.Context, name string, client clientv1.EventInterface, clusterClient rayiov1alpha1.RayClusterInterface) ([]v1.Event, error) {
+	rayCluster, err := getClusterByName(ctx, clusterClient, name)
```

Comment:
let's handle the err here. Otherwise, it could leads to panic issue (rayCluster = nil)